### ISSUE TITLE
5.3 Παραστατικά

### ISSUE THREAD

**Issue by sadamaki** [mou-gr/regions#116]
Η δαπάνη "MARKETING & Distribution, ΒΆΣΕΙ ισχύοντος τεχνικού παραρτήματος είναι 3.000, αν ο χρήστης αλλάξει με το χέρι το ποσό της σε πχ  3.200  κατά την αντιστοίχηση της  με το τιμολόγιο αγοράς τότε, waring μήνυμα :

κΑΙ ΌΧΙ υποχρεωτικό μήνυμα) ΜΗΝΥΜΑ ΣΤΟΝ ΠΊΝΑΚΑ ΠΑΡΑΣΤΑΤΙΚΩΝ ΑΓΟΡΑΣ : 

"Βρέθηκαν δαπάνες όπου το συνολικό αντιστοιχισμένο ποσό, υπερβαίνει το συνολικό ποσό της δαπάνης:
-px  Δαπάνη MARKETING & Distribution,  ": Συνολικό ποσό δαπάνης [3.000,00 €], αντιστοιχισμένο ποσό [3.200,00 €].

**Comment by galanosel** [comment]
Προς το παρόν έχουν υλοποιηθεί έλεγχοι σχετικά με αυτό στον έλεγχο ορθότητας.
Σε επόμενη έκδοση θα βάλουμε έλεγχο και εντός της φόρμας καταχώρησης

**Comment by sadamaki** [comment]
Φαίνεται στον 5.2 πίνακα να έχει αμοιβές συμβούλων 83.500 , ενώ με το τιμολόγιο αγοράς 15623 για τις αμοιβές συμβούλων αν δεις έχει δηλώσει 84.000 και στον πίνακα PurchaseVoucherDetails_Grid
έχω αλλάξει το ποσό της στήλης αξίας μονάδας σε 84.000 και έχω κάνει"χρήση" .

Στον έλεγχο ορθότητας όπως λες δεν εμφανίζεται κανένα σχετικό μήνυμα.

**Comment by sadamaki** [comment]
Ναι, όμως δες τώρα που ξαναγύρισα στην αρχική την αξία δαπάνης αμοιβών συμβούλων σε 83500 με παραστατικό αγοράς 83500, εξακολουθεί και εμφανίζεται πάλι ο έλεγχος ορθότητας 

[5.2 Κατηγορία Δαπάνης: 2 - Αμοιβές συμβούλων (Δικαιούχος: ΑΔΑΜΑΚΗ ΕΞΩ, Συνολικό (€): 83500,00)] Το συνολικό ποσό των δαπανών που έχουν συνδεθεί με την κατηγορία δαπάνης υπερβαίνει το εγκεκριμένο ποσό αυτής

**Comment by sadamaki** [comment]
Λευτέρη, δεν ξέρω αν πρέπει να υλοποιηθεί με έλεγχο ορθότητας. Ας το δει και ο ΕΦΕΠΑΕ, διότι σε παλιές δράσεις ήταν όπως σου αναφέρω εξαρχής, ήταν ήταν μέσα στη φόρμα καταχώρησης, είτε μόνο με warning μήνυμα στον έλεγχο ορθότητας (και όχι υποχρεωτικό μήνυμα που υλοποίησες προς το παρόν)

**Comment by sadamaki** [comment]
Μάλλον θα χρειαστεί να επανέλθουμε στην αρχική υλοποίηση που ζήτησα να είναι warning (KAI όχι υποχρεωτικοί) οι παραπάνω έλεγχοι ορθότητας (όταν δλδ το συνολικό ποσό των δαπανών που έχουν συνδεθεί με δαπάνη υπερβαίνει το εγκεκριμένο ποσό αυτής).
Το αναφέρω στο από 16/8 και 17/9 issue παραπάνω.
